index.js

```diff
@@ -64,8 +68,13 @@ yargs
     .option('vanilla',{
         desc: 'create blank plain JS project',
         type: 'boolean',
+        default: true
```

Comment:
don't make this true. Keep this false.

Reply:
We should sync up with @amgando . There is a good argument for making the default project to be the simplest possible in terms of code base. I'll change it back until we can sync up